content/workflow.md

```diff
@@ -0,0 +1,406 @@
+
+Design the steps of the workflow and create an order of events. Execute the functions (work) inside the steps. Get the result after sequential processing of steps or functions. You may check the results of individual steps as well.
+
+Hypi provides below given `Workflow` data type for the implementation of the sequence of steps.
+```java
+type Workflow {
+    name: String!
+    cronSchedule: String
+    execAs: String
+    async: Boolean
+    parallel: Boolean
+    maxExecutionTime: String
+    steps(...): [WorkflowStep!]
+}
+```
+The parameters are as follows:
+
+
+| Parameter        | Type         | Description                                                                                                                                                                                                                                                                                               |
+|------------------|--------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Name             | String       | Name of the workflow                                                                                                                                                                                                                                                                                      |
+| cronSchedule     | String       | If present, this is a \`cron\` schedule to automatically execute this Workflow The syntax as defined at https://www.manpagez.com/man/5/crontab/ NOTE: The special strings @hourly, @daily, etc are NOT supported                                                                                          |
+| execAs           | String       | An ArcQL query to find the account e.g. hypi.id = 'user123' to find by id or username = 'blah' to find by username.<br><br>If present, execution of the steps in the Workflow will be done as this account. If not specified, it defaults to the account making the request                               |
+| Async            | Boolean      | Set to \`true\` for asynchronous processing                                                                                                                                                                                                                                                               |
+| Parallel         | Boolean      | If present AND true, all steps in the workflow are executed at the same time.                                                                                                                                                                                                                             |
```

Comment:
`parallel` - the uppercase matters

Reply:
The first letters in the table were auto corrected with capital letters. Will change